### PR TITLE
update panic hook for zebra-test to supress confusing output

### PR DESCRIPTION
**Pull request by yaahc**
this prevents tests that return an `Err(Report)` from printing the content of the report followed by a verbose panic message explaining that the test returned an `Err`

### CODE REVIEW

File: zebra-test/Cargo.toml [hunk 1]
```diff
@@ -21,6 +21,7 @@ regex = "1.3.9"
 thiserror = "1.0.20"
 tempdir = "0.3.7"
 pretty_assertions = "0.6.1"
+owo-colors = "1.1.3"
```
Comment (hdevalence) [inline zebra-test/Cargo.toml:24]: owo whats this?  https://crates.io/crates/owo-colors

Comment (hdevalence) [reply zebra-test/Cargo.toml:24]: looks neat, no objection to adding this to test code

**Review by hdevalence — APPROVED** [review]
:ship:

File: zebra-test/Cargo.toml (hunk 1 above)
Comment (yaahc) [reply zebra-test/Cargo.toml:24]: > owo whats this?

:rofl:

**Pull request merged**